dss/events/handlers/index.py

```diff
@@ -76,24 +95,34 @@ def create_index_data(s3, bucket_name, manifest, logger):
             # of the query spec, so go with that, at least for now. Do so by substituting
             # dot for underscore in the key filename portion of the index.
             # As due diligence, additional investigation should be performed.
-            index_filename = file_info["name"].replace(".", "_")
+            index_filename = file_info[BundleFileMetadata.NAME].replace(".", "_")
             index_files[index_filename] = file_json
     index['files'] = index_files
     return index
 
 
+def get_bundle_id_from_key(bundle_key):
+    if bundle_key.startswith("bundles/"):
+        bundle_key = bundle_key[8:]
+    return bundle_key
+
+
 def create_file_key(file_info) -> str:
```

Comment:
this abstraction probably belongs elsewhere, since it's used in multiple places.